helmfile/values/prometheus-blackbox-exporter-sc.yaml.gotmpl

```diff
@@ -2,21 +2,55 @@ resources: {{- toYaml .Values.prometheusBlackboxExporter.resources | nindent 4 }
 
 config:
   modules:
-    http_401:
+    http_400:
```

Comment:
There are other probes further down that were using the `http_401` module. So please add that back so that we have one for 400 and one for 401.